RecoLocalCalo/HGCalRecAlgos/interface/HGCal3DClustering.h

```diff
@@ -0,0 +1,88 @@
+#ifndef RecoLocalCalo_HGCalRecAlgos_HGCal3DClustering
+#define RecoLocalCalo_HGCalRecAlgos_HGCal3DClustering
+
+
+#include "DataFormats/Math/interface/Point3D.h"
+#include "DataFormats/EgammaReco/interface/BasicCluster.h"
+#include "DataFormats/ParticleFlowReco/interface/HGCalMultiCluster.h"
+
+#include <list>
```

Comment:
This include does not appear to be needed
Includes for `<vector>` and `<array>` should be added